shared/team-building/shared.tsx

```diff
@@ -38,6 +38,11 @@ const services: {[K in ServiceIdWithContact]: {color: string; icon: IconType; la
     icon: 'iconfont-identity-pgp',
     label: 'PGP',
   },
+  phone: {
+    color: '#4C8EFF',
+    icon: 'iconfont-phone-contact',
```

Comment:
@cecileboucheron Do you know the name of the correct icon to use per the design? I don't see the matching icon in `shared/images/iconfont/`.